src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -258,6 +268,9 @@ export default {
 		}
 	},
 	methods: {
+		determineIsMobile() {
```

Comment:
We do have this `isMobile` pattern elsewhere in the code, so no need to change for this PR, but for future mobile-only UI (CSS/text), consider using `@media`. That would avoid an extra event listener.

Reply:
Yeah I thought about it but I think we needed this for conditionally rendering some elements (different text displayed on mobile), unless there is a pure css way to do this 🤔

Reply:
Yeah, you can add a class `md:tw-hidden` to every element you want to show on mobile only, this will be hidden for medium sizes or larger